firmware/Core/Src/telecommands/agenda_telecommands.c

```diff
@@ -0,0 +1,21 @@
+#include "telecommands/telecommand_definitions.h"
+#include "telecommands/telecommand_args_helpers.h"
+#include "telecommands/telecommand_executor.h"
+#include "debug_tools/debug_uart.h"
+#include "telecommands/agenda_telecommands.h"
+
+#include <string.h>
+#include <stdio.h>
+
+/// @brief Telecommand: Fetch all active agendas
```

Comment:
Change to "Fetch all pending agenda items, and log them each as JSONL"

Reply:
Done